contrib/completion/bash/docker

```diff
@@ -1788,17 +1788,17 @@ _docker_run() {
 			;;
 		--security-opt)
```

Comment:
Sorry David, this change does not work: `=` cannot be substituted for `:` here because it is in `COMP_WORDBREAKS`.
I will create a fix.